tests/unittests/test_log_filtering_functions.py

```diff
@@ -20,6 +19,15 @@
 
 
 class TestLogFilteringFunctions(testutils.WebHostTestCase):
+    """This class is for testing the logger behavior in Python Worker when
+    dealing with customer's log and system's log. Here's a list of expected
+    behaviors:
```

Comment:
nit; you don't need to mention this - if you have mentioned this earlier. Just give a reference of `is_system_log_category` and that should be good enough.